File: ipman.shared/Entity/Tag.cs
using ipman.shared.Entity.Join;
using System;
using System.Collections.Generic;
using System.Text;

namespace ipman.shared.Entity
{
    public class Tag : EntityBase
    {
        public string TagName { get; set; }
        public string TagImage { get; set; }
        public Guid SiteAccountID { get; set; }
        public SiteAccount SiteAccount { get; set; }
        public bool IsActive { get; set; }
        public DateTime CreatedUTC { get; set; }
        public virtual ICollection<PostTag> PostTags { get; set; }
    }
}

File: ipman/ClientApp/entity/Tag.ts
import PostTag from "./join/PostTag";
import { EntityBase } from "./EntityBase";
import SiteAccount from "./SiteAccount";

export default interface Tag extends EntityBase {
    TagName: string;
    TagImage: string;
    SiteAccountID: string;
    SiteAccount: SiteAccount;
    IsActive: boolean;
    CreatedUTC: Date | string;
    PostTags: PostTag[];
}
